internal/policies/dconf/dconf.go

```diff
@@ -1,4 +1,28 @@
 // Package dconf is the policy manager for dconf entry types.
+//
+// The manager will create additional dconf databases: system-db:<username> and system-db:machine.
+// Values specified in the GPO will be appended to those databases, according to their configuration.
+// Dconf applies the values from bottom to top and stops checking values for a certain key at the
+// moment it finds a lock.
+// Default values specified by the policy will be added to the profile database, along with locks to
+// their correspondent keys, in order to enforce the requested values.
+//
+// The manager will parse the values and try to fix some formatting problems, but if something goes
+// wrong when applying the profile or updating dconf, authentication is prevented.
```

Comment:
I would say, instead of "authentication is prevented", because we know this is the general behavior of a policy manager when we return an error (and it’s handled above in the stack), that we return an error (the consequence is implicit).